FirebaseInstallations/Source/Library/FIRInstallationsItem.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Add short docs to the API
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSInteger, FIRInstallationsStatus);
+
+@class FIRInstallationsStoredItem;
+@class FIRInstallationsStoredAuthToken;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRInstallationsItem : NSObject
+
+@property(nonatomic, readonly, nonnull) NSString *appID;
```

Comment:
do you still need this now that you have the nonnull macro on top?

Reply:
Good catch. I'll update the properties attributes